samples/modules/lvgl/acceleration_chart/src/main.c

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2023 Benjamin Cabé <benjamin@zephyrproject.org>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/display.h>
+#include <zephyr/drivers/sensor.h>
+
+#include <lvgl.h>
+#include <stdio.h>
+#include <string.h>
+#include <zephyr/kernel.h>
+
+#define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(app);
+
+static lv_obj_t *chart1;
+static lv_chart_series_t *ser_x;
+static lv_chart_series_t *ser_y;
+static lv_chart_series_t *ser_z;
+lv_timer_t *fetch_and_display_timer;
```

Comment:
Is there any reason you did not make this one static?

Reply:
+1

Reply:
done and renamed to `sensor_timer`, also renamed the callback to `sensor_timer_cb` (again, I went ffor the convention used in the demos)